src/components/BooksItem.jsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
```

Comment:
### Although I am highlighting this piece of code, the change I am about to request should be done in the `src/components/BooksList.js` file. 

- [ ] Kindly use the `useSelector` hook to access the store and retrieve the books reducer in that file. This means that you would have to remove the **_static array_** you created in that file and replace it with the `initialState` array gotten from the books reducer in the redux slice file for the books. Kindly implement this change. 👍 💻 

#### Hint: 
```JavaScript
import { useSelector } from 'react-redux';
const books = useSelector((state) => state.book.bookItem);
```

Reply:
thank you so much resoved